README.md

```diff
@@ -1,2 +1,91 @@
 # queries
+
```

Comment:
general comment: github PRs don't save patchsets, so please add another commit instead of --amend, because all history is just lost. When merging just squash it.

Reply:
I agree and I will try, but I cannot guarantee, is quite hard to change muscle-memory.